build.sbt

```diff
@@ -66,6 +67,7 @@ libraryDependencies ++= Seq(
     //"org.scalatra.scalate" %% "scalate-core" % scalateVersion,
     "com.typesafe" % "config" % typesafeVersion,
     "io.spray" %% "spray-json" % sprayVersion,
+    "com.nrinaudo" %% "kantan.csv" % katanVersion,
```

Comment:
I see that you've added real CSV reader, that is good because we will be able to read complex CSV files. However, it has to be carefully measured because it is an external library that we didn't write. You need to consider the following factors: 
1) Is it secure
2) Who maintains it? Can we trust them to do a good job? 
3) Is it a big jar file that has lots of stuff we don't need? 

The alternative is to use the trivial CSV parsing that we had, and call it a 20/80 solution.

Reply:
This is a good point, but I think it is a question that applies to all the libraries we depend on. We should create guidelines for using 3rd party libraries.